grails-app/controllers/org/pih/warehouse/inventory/StockTransferController.groovy

```diff
@@ -97,4 +98,34 @@ class StockTransferController {
         }
         redirect(action: "list")
     }
+
+    def deleteInboundOrOutboundReturn = {
```

Comment:
deleteStockTransfer

Reply:
Make sure it is only for superusers, otherwise move it to the `eraseStockTransfer`